modules/kazoo/kz_amqp.c

```diff
@@ -684,8 +684,6 @@ int kz_amqp_add_connection(modparam_t type, void* val)
 
     if(newConn->info.vhost == NULL) {
     	newConn->info.vhost = KZ_URL_ROOT;
-    } else if(newConn->info.vhost[0] == '/' && strlen(newConn->info.vhost) == 1) { // bug in amqp_parse_url ?
-    	newConn->info.vhost++;
```

Comment:
But this prevents to use rabbitmq-c lib 0.8.0
With this strings kazoo module cannot connect to rabbitmq-server